packages/codemod/transforms/__testfixtures__/react-loadable-to-loadable-component_arrow-w-params.output.js

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable */
+import loadable from '@loadable/component'
+
+const CustomLinkLoadable = loadable(() =>
+  import(/* webpackChunkName: "custom-link" */ '@components/CustomLink/Link'), {
+  fallback: null,
```

Comment:
it should throw in this case, as long as some user logic is going to be lost.

Reply:
Ah, I am having quite a bit of dilemma with that one. I assume if we were to throw in that case, the codemod would be quite unusable. 

I do not have a number, but I assume the number of people using that pattern should be quite big, especially because it's in the [README.md](https://github.com/jamiebuilds/react-loadable#timing-out-when-the-loader-is-taking-too-long) of the project. 🤔 

Would be happy to get additional perspective regarding this one!

Reply:
Would it make sense to solve this by extending `loadable-components` so that the fallback can be either an element (like today) or a render function/component, like `react-loadable`? 

Something like `React.isValidElement(fallback) ? fallback : React.createElement(fallback, props)` should be enough. I guess the problem might be figuring out whatever `props` should be?

Reply:
Since `react-loadable` has some behaviours that don't exist in `@loadable/component`, I am not sure if there is a way to make the change without changing some logics; unless of course `@loadable/component` adds the same APIs that implement the same behaviours.

Reply:
Yeah, I get that. A sort of halfway there solution could be to wrap the inline function/loading component in a `React.createElement` call with no props. This would at least render the default state and preserve the existing code. 

The transform should probably still write something to the console about that loading components aren't fully supported and that additional action might be needed.

Reply:
Just added it! Please kindly review it again 🙂